cli/slsa-verifier/main_regression_test.go

```diff
@@ -1510,6 +1510,66 @@ func Test_runVerifyGHAContainerBased(t *testing.T) {
 	}
 }
 
+func Test_runVerifyBCRModule(t *testing.T) {
+	t.Parallel()
+	os.Setenv("SLSA_VERIFIER_EXPERIMENTAL", "1")
```

Comment:
Use `t.Setenv` so that the environment variable is restored automatically up after the test. Do not use `t.Parallel` as it will cause race condition with the environment variables. See [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv)
```suggestion
	t.Setenv("SLSA_VERIFIER_EXPERIMENTAL", "1")
```